### DISCUSSION TITLE
Figma o Adobe XD?

### DISCUSSION THREAD

**Discussion by ElPoder1** [Launch-X-Latam/MisionFrontEnd#51] [Q&A]
Leyendo el contenido de FrontEnd me encontré con estas herramientas para trabajar, pero recientemente salió la **noticia** donde **Adobe esta en proceso de comprar a Figma**, independientemente de esto, _con cual herramienta recomendarían trabajar?_  o que sugerencias recomiendan

**Comment by doguedogue** [comment]
Figma es genial, mira este tutorial te lleva de la mano en como hacer desde 0 el diseño

https://www.youtube.com/watch?v=c9Wg6Cb_YlU&t=305s

**Comment by maxkario** [comment]
Tranquilo, aun no se ha concretado la compra, ese proceso es largo. Mientras puedes usar Figma y en todo caso, como lo indican en el tema, puedes usar la version gratis Adobe XD

**Comment by AlvaroFierro** [comment]
Yo en lo personal uso mucho Figma porque es muy facil de usar y no necesitas llegar a un nivel tan profesional antes de poder usarlo. Pero ahora si que es con el que te sientas mas comodo @ElPoder1

**Comment by gjordanra** [comment]
Aun no vamos por ahi, pero lo que más se ve que la gente usa y  a veces piden en los empleos es Figma

**Comment by AoGnyan** [comment]
Webflow de una

**Comment by koshishark** [comment]
Dudo que la compra de Adobe afecte en algo, salvo algunas funcionalidades de pago que agreguen, quizá, pero prefiero Figma. Yo lo aprendí porque la mayoría de los tutoriales que encuentras son orientados a Figma (y, pues, así aprendí a utilizarlo), y en muchas empresas trabajan con ella. Mi recomendación es que pruebes ambas y trabajes con la que te sientas más a gusto.

**Comment by FernandoPeraltaCarlos** [comment]
Figma , por ahora, con su compra por parte de adobe quien sabe que tanto cambie
